src/joLinearFunctionConstructors/joExtend.jl

```diff
@@ -187,3 +187,9 @@ function joExtend(n::Integer,pad_type::Symbol;
     return nothing
 end
 
+function joExtend(n::NTuple{N, Integer},pad_type::Symbol;
```

Comment:
That's not the convention in JOLI, the convention is `joXXX(n1, n2, n3 ....)` no a tuple.

Reply:
Thanks. `Vararg{Integer, N}` would only be OK if that is the last argument. However, there is a `pad_type` after that. Any suggestion?